src/Streamly/Internal/Data/Fold/Types.hs

```diff
@@ -644,15 +644,15 @@ instance (Monad m, Floating b) => Floating (Fold m a b) where
 -- Internal APIs
 ------------------------------------------------------------------------------
 
--- | @(lmap f fold)@ maps the function @f@ on the input of the fold.
+-- | @(map f fold)@ maps the function @f@ on the input of the fold.
```

Comment:
We should emphatically note that this is not the same as `fmap`. `fmap` is covariant while this contravariant.